### ISSUE TITLE
VRS: Focal loss

### ISSUE THREAD

**Issue by torbjorgen** [Clinical-Genomics/rdds#300]
Using focal loss provides validation set improvement.

`tf.keras.losses.binary_focal_crossentropy(y_true=y_true, y_pred=y_pred, from_logits=False)`

Use and retrain model.
